File: VB/WinWebSolution.Module.Web/WebSortListViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Web.Editors.ASPx

Namespace WinWebSolution.Module.Win
	Public Class WinSortRootListViewController
		Inherits SortListViewControllerBase
		Protected Overrides Sub OnViewControlsCreated()
			MyBase.OnViewControlsCreated()
			Dim gridListEditor As ASPxGridListEditor = TryCast(View.Editor, ASPxGridListEditor)
			If gridListEditor IsNot Nothing Then
				gridListEditor.Grid.SettingsBehavior.AllowSort = False
			End If
		End Sub
	End Class
End Namespace

File: CS/WinWebSolution.Module.Web/WebSortListViewController.cs
using DevExpress.ExpressApp.Web.Editors.ASPx;

namespace WinWebSolution.Module.Win {
    public class WinSortRootListViewController : SortListViewControllerBase {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            ASPxGridListEditor gridListEditor = View.Editor as ASPxGridListEditor;
            if (gridListEditor != null) {
                gridListEditor.Grid.SettingsBehavior.AllowSort = false;
            }
        }
    }
}